ui/slider.js

```diff
@@ -410,9 +410,24 @@ return $.widget( "ui.slider", $.ui.mouse, {
 		}
 	},
 
-	_setOption: function( key, value ) {
+	_changeAllValues: function( valuesLength ) {
 		var i,
-			valsLength = 0;
+			rangeValues = [ this.values( 0 ), this.values( 1 ), this._valueMax() ];
+
+		if ( this.options.range === true && $.unique( rangeValues ).length === 1 ) {
```

Comment:
Yes, I've also tested that, it works well. I was thinking that setting values with reverse loop to the minimum will lock slider, but didn't check it.